sdk/confidentialledger/azure-security-confidentialledger/src/test/java/com/azure/security/confidentialledger/GetEnclaveQuotesTests.java

```
@@ -30,46 +18,20 @@ public final class GetEnclaveQuotesTests extends ConfidentialLedgerClientTestBas
-        reactor.netty.http.client.HttpClient reactorClient = reactor.netty.http.client.HttpClient.create()
-                .secure(sslContextSpec -> sslContextSpec.sslContext(sslContext));
-        HttpClient httpClient = new NettyAsyncHttpClientBuilder(reactorClient).wiretap(true).build();
-
-        if (getTestMode() == TestMode.PLAYBACK) {
-            confidentialLedgerClientBuilder
-                .httpClient(interceptorManager.getPlaybackClient())
-                .credential(request -> Mono.just(new AccessToken("this_is_a_token", OffsetDateTime.MAX)));
-        } else if (getTestMode() == TestMode.RECORD) {
-            confidentialLedgerClientBuilder
-                .addPolicy(interceptorManager.getRecordPolicy())
-                .httpClient(httpClient)
-                .credential(new DefaultAzureCredentialBuilder().build());
-        } else if (getTestMode() == TestMode.LIVE) {
-            confidentialLedgerClientBuilder
-                .credential(new DefaultAzureCredentialBuilder().build())
-                .httpClient(httpClient);
-        }
-        
-        ConfidentialLedgerClient client = confidentialLedgerClientBuilder.buildClient();
-        Response<BinaryData> enclaveQuotesWithResponse = client.getEnclaveQuotesWithResponse(null);
+        Response<BinaryData> enclaveQuotesWithResponse = confidentialLedgerClient.getEnclaveQuotesWithResponse(null);
         Assertions.assertEquals(enclaveQuotesWithResponse.getStatusCode(), 200);
         BinaryData parsedResponse = enclaveQuotesWithResponse.getValue();
 
         ObjectMapper objectMapper = new ObjectMapper();
-        JsonNode responseBodyJson = objectMapper.readTree(parsedResponse.toBytes());
+        JsonNode responseBodyJson = null;
+
+        try {
+            responseBodyJson = objectMapper.readTree(parsedResponse.toBytes());
+        } catch (IOException e) {
+            e.printStackTrace();
+            Assertions.assertTrue(false);
+        }
```

This is not required, just let the exception bubble up. It'll fail the test and print the stacktrace.